clinica/iotools/converters/ixi_to_bids/ixi_to_bids.py

```diff
@@ -0,0 +1,70 @@
+"""Convert OASIS dataset (https://sites.wustl.edu/oasisbrains/) to BIDS."""
+
+from pathlib import Path
+from typing import Optional
+
+import nibabel as nb
+import numpy as np
+
+from clinica.iotools.bids_utils import write_modality_agnostic_files
+from clinica.iotools.converters.ixi_to_bids.ixi_to_bids_utils import *
```

Comment:
Usually it is recommended to avoid this kind of * import.
Prefer either importing the module (and prefixing all objects with the module name):

```python
from clinica.iotools.converters.ixi_to_bids import ixi_to_bids_utils
....
ixi_to_bids_utils.define_participants(...)
```

, or importing explicitly what you need:

```python
from .ixi_to_bids_utils import (
    read_ixi_clinical_data,
    define_participants,
   ...
)
```

For simple cases where there is no name conflicts, I tend to use the second option.